tests/testthat/test-usetemplate.R

```diff
@@ -95,6 +100,12 @@ test_that("Scatter plot brand names",
     res <- GetVectorOfColors(tmp.template, scatter.dat, NULL, "Scatter", 4,
                 palette = "Default or template settings")
     expect_equal(res, checkColors(col.vec))
+    
```

Comment:
Can there be a test that verifies that the correct colours are returned by `GetVectorOfColors` in this case?

Reply:
Added a check for the return values in ea08b0d